### ISSUE TITLE
add splash screen

### ISSUE THREAD

**Issue by TtedyT** [IdoSagiv/CommuniDog#57]
* Will improve UX: user don't have to see frozen screen while things load(db requests etc.)
* The UX improvment will help the app's number of users grow
* Will make the app look professional: most of the apps have splash screen

read more:
- [What is Splash Screen & How to Use](https://www.spinxdigital.com/blog/use-splash-screens-the-right-way/)
- [30 inspiring splash screen designs](https://www.justinmind.com/blog/splash-screen-designs/)
- [splash screen android developers](https://developer.android.com/about/versions/12/features/splash-screen)
- [Splash Screen - Android Studio Tutorial](https://www.youtube.com/watch?v=Q0gRqbtFLcw)

**Comment by IdoSagiv** [comment]
Nice idea! Can i assign you the job?

**Comment by TtedyT** [comment]
hi in primitive app implementations the developer freezes the home screen until the wanted data is downloaded from db,
in out previous implementation we didn't even do this, instead - we called some db methods - then imminently moved to the next screen.
The better implementation is to use OnSuccess listener to know when the db finished downloading the wanted data.
IN CURRENT IMPLEMENTATION - we added progress bar to the ui BUT it seems that we still move to the next screen without using OnSuccess.
